AppController/lib/helperfunctions.rb

```diff
@@ -478,6 +478,13 @@ def self.app_has_config_file?(tar_gz_location)
     end
   end
 
+
+  # Prepare the application code to be run by AppServers.
+  #
+  # Args:
+  #   app_name: A String containing the application id.
+  # Returns
+  #   A Boolean indicating if the setup was successful.
```

Comment:
Returning a boolean as an indicator of success seems strange to me here. I would recommend throwing an exception when there is a failure.